content/docs/next/branch_management.md

```diff
@@ -25,7 +25,7 @@ Before the release of the next stable version, feature PRs will be frozen. A [re
 
 All branches with prefix `release-` are considered _stable_ branches.
 
-After every minor release (http://semver.org/), we will have a new stable branch for that release, managed by a [patch release manager](./dev-internal/release#release-management). We will keep fixing the backwards-compatible bugs for the latest two stable releases. A _patch_ release to each supported release branch, incorporating any bug fixes, will be once every two weeks, given any patches.
+After every minor release (http://semver.org/\), we will have a new stable branch for that release, managed by a [patch release manager](./dev-internal/release#release-management). We will keep fixing the backwards-compatible bugs for the latest two stable releases. A _patch_ release to each supported release branch, incorporating any bug fixes, will be once every two weeks, given any patches.
```

Comment:
Why the `\` at the end of `http://semver.org/\`?

Reply:
Without the `\` Hugo renders the link with the `),` appended.
* https://etcd.io/docs/next/branch_management/#stable-branches
* https://deploy-preview-203--etcd.netlify.app/docs/next/branch_management/#stable-branches

Reply:
Oh, that's a raw URL! (I was reviewing too quickly.) We should avoid raw URLs in the docs. Hmm, I thought that I had fixed that particular case -- maybe on the `docsy` branch? I'll check tomorrow, or as soon as I can.

Reply:
> We should avoid raw URLs in the docs.

I agree.

Reply:
Will track via #253